pkg/deps/resolve_test.go

```diff
@@ -761,6 +761,21 @@ spec:
 			},
 			expErrSubstr: "no latest version",
 		},
+		{
+			desc:     "annotations: bad fixed version",
+			universe: annotationSet,
+			comps: []bundle.ComponentReference{
+				{ComponentName: "ann", Version: "1.1.0"},
```

Comment:
This doesn't clearly express that component 'ann' is annotated with 'bad-component'. Can you change naming to make that more explicit or create a new component?

Reply:
Done